legacy/audit/auditor.go

```diff
@@ -64,15 +64,14 @@ func InitRealServerContext(
 			ReadRepo:         reg.MkReadRepositoryCmdReal,
 			ReadManifestList: reg.MkReadManifestListCmdReal,
 		},
-		VerboseLogging: verbose,
 	}
 
 	return &serverContext, nil
 }
 
 // RunAuditor runs an HTTP server.
 func (s *ServerContext) RunAuditor() {
-	s.Debug("Running on Process ID (PID):", os.Getgid())
+	logclient.Debug("Running on Process ID (PID):", os.Getgid())
```

Comment:
Adding on the other comment, Debug should probably be an interfact method and be implemented by concrete loggers.